packages/yourloops/pages/caregiver/patients/page.tsx

```diff
@@ -257,8 +257,7 @@ function PatientListPage(): JSX.Element {
   };
 
   React.useEffect(() => {
-    document.title = `${t("my-patients-title")} - ${t("brand-name")}`;
-    log.info("Set document title to", document.title);
+    setPageTitle(t("my-patients-title"));
```

Comment:
the spec says that it should `Patients (string to translate) | YourLoops` but I thing it's better with what you set. I updated the spec accordingly